thewalrus/_permanent.py

```diff
@@ -15,29 +15,24 @@
 Permanent Python interface
 """
 import numpy as np
-
+from numba import jit
 from ._hafnian import hafnian_repeated
-from .libwalrus import perm_complex, perm_real, perm_BBFG_real, perm_BBFG_complex
 
+# pylint: disable = C0103, R0914
```

Comment:
where was C0103 (invalid name) coming from?

Reply:
It was coming from our variables' names in the beginning, but I just tried pylint without any disables and it gave me 10.00/10